packages/integration/lib/src/views/post/post_detail_screen.dart

```diff
@@ -126,6 +133,7 @@ class _LMFeedPostDetailScreenState extends State<LMFeedPostDetailScreen> {
         return value;
       });
     }
+    config = widget.config ?? LMFeedConfig().postDetailConfig;
```

Comment:
This will break because we are not using passed configs from client side

Reply:
done